localization/pose_initializer/src/pose_initializer/pose_initializer_core.cpp

```diff
@@ -97,7 +106,7 @@ geometry_msgs::msg::PoseWithCovarianceStamped PoseInitializer::get_gnss_pose()
 {
   if (gnss_) {
     PoseWithCovarianceStamped pose = gnss_->get_pose();
-    pose.pose.covariance = gnss_particle_covariance_;
+    if (ndt_) pose.pose.covariance = gnss_particle_covariance_;
```

Comment:
What is the reason for overwriting only NDT?

Reply:
I thought this process was unnecessary when using gnss/imu because the subscribed gnss/imu topics have already been given covariance.

Reply:
I see. It seems better to move this processing to the GNSS driver. I will make a PR later, so could you please delete it for now?

Reply:
Deleted
https://github.com/autowarefoundation/autoware.universe/pull/2904/commits/6a98d46c1a320815ad9b82f8073750eb121663d8